doc/docs.md

```diff
@@ -2206,6 +2187,12 @@ button.Size = Size{
 }
 ```
 
+If multiple embedded structs have methods or fields with the same name, or if methods or fields
+with the same name are defined in the struct, you can call methods or assign to variables in
+the embedded struct like `button.Size.area()`.
+When you do not specify the embedded struct name, the method of the outermost embedded 
```

Comment:
```suggestion
When you do not specify the embedded struct name, the method of the outermost 
```

Reply:
The struct you're embedding into may have a duplicate for an embedded field.  It will be used first, not the first embedded struct.

Reply:
```v
struct FooA{
	x int
}

struct FooB{
	FooA
	x int
}

struct FooC{
	FooB
	x int
}

struct FooD{
	FooC
}

struct Bar{
	FooD
}

fn main(){
	b := Bar{x:10}
	dump(b)
}
```
```
Bar{
    FooD: FooD{
        FooC: FooC{
            FooB: FooB{
                FooA: FooA{
                    x: 0
                }
                x: 0
            }
            x: 10
        }
    }
}
```
This is the case I envisaged, how would you describe it?
I think this sentence is not enough.

Reply:
Personally, I think it would be better in this case if the x of all embedded structures were set to a value.
This is a matter of specification and needs to be discussed.